src/patternfly/components/Content/styles.scss

```diff
@@ -1,36 +1,135 @@
+  --pf-c-content--FontWeight: var(--global--FontWeight--normal);
+
+  // Small text
+  --pf-c-content--small--MarginBottom: var(--global--spacer);
+  --pf-c-content--small--LineHeight: var(--global--LineHeight--lg);
+  --pf-c-content--small--FontSize: var(--global--FontSize--sm);
+  --pf-c-content--small--Color: var(--global--Color--dark-alt);
+
+  // Links
+  --pf-c-content--a--Color: var(--global--link--Color);
+  --pf-c-content--a--TextDecoration: var(--global--link--TextDecoration);
+  --pf-c-content--a--hover--Color: var(--global--link--Color--hover);
+  --pf-c-content--a--hover--TextDecoration: var(--global--link--TextDecoration--hover);
+
+  // Blockquote
+  --pf-c-content--blockquote--Padding: var(--global--spacer);
+  --pf-c-content--blockquote--FontWeight: var(--global--FontWeight--light);
+  --pf-c-content--blockquote--Color: var(--global--Color--dark-alt);
+  --pf-c-content--blockquote--BorderLeftColor: var(--global--BorderColor--light);
+  --pf-c-content--blockquote--BorderLeftWidth: var(--global--BorderWidth--md);
+
+  // Lists
+  --pf-c-content--ul--PaddingLeft: var(--global--spacer--md);
+  --pf-c-content--ol--MarginTop: var(--global--spacer);
+  --pf-c-content--ol--MarginLeft: var(--global--spacer--md);
+  --pf-c-content--ul--MarginTop: var(--global--spacer);
+  --pf-c-content--ul--MarginLeft: var(--global--spacer--md);
+  --pf-c-content--ul--nested--MarginTop: var(--global--spacer--sm);
+  --pf-c-content--ul--nested--MarginLeft: var(--global--spacer--sm);
+  --pf-c-content--ul--ListStyle: var(--global-ListStyle);
+  --pf-c-content--li--MarginTop: var(--global--spacer--sm);
+  --pf-c-content--dd--MarginBottom: var(--global--spacer--sm);
+  --pf-c-content--dd--MarginLeft: var(--global--spacer--md);
+}
+
+// stylelint-enable
+
 .pf-c-content {
+  font-size: var(--pf-c-content--FontSize);
+  line-height: var(--pf-c-content--LineHeight);
+
   a {
-    color: $global--link--Color;
-    text-decoration: $global--link--TextDecoration;
+    color: var(--pf-c-content--a--Color);
+    text-decoration: var(--pf-c-content--a--TextDecoration);
 
     &:hover {
-      color: $global--link--Color--hover;
-      text-decoration: $global--link--TextDecoration--hover;
+      color: var(--pf-c-content--a--hover--Color);
+      text-decoration: var(--pf-c-content--a--hover--TextDecoration);
     }
   }
 
   li + li {
-    margin-top: $global--spacer--sm;
+    margin-top: var(--pf-c-content--li--MarginTop);
   }
 
   p,
   dl,
   ol,
   ul,
   blockquote,
+  small,
   pre,
   table {
-    &:not(:last-child){
-      margin-bottom: $global--spacer--lg;
+    &:not(:last-child) {
+      // This variable name doesn't reflect the selector, it's an excpection to the variable system.
+      margin-bottom: var(--pf-c-content--MarginBottom);
     }
   }
 
-  dl,
   ol,
   ul {
-    padding-left: $global--spacer--lg;
+    // This variable name doesn't reflect the selector, it's an excpection to the variable system.
```

Comment:
the indent on these two lines are causing the linter to fail

Reply:
fixed it